.eslintrc.cjs

```diff
@@ -9,7 +9,9 @@ module.exports = {
 	],
 	rules: {
 		// blocked by https://github.com/sveltejs/eslint-plugin-svelte/issues/652
-		'svelte/valid-compile': 'off'
+		'svelte/valid-compile': 'off',
+		'svelte/no-at-html-tags': 'off',
```

Comment:
added content policy requirements to avoid issues: https://kit.svelte.dev/docs/configuration#csp

Reply:
JK it became a mess I didn't

Reply:
rewrote this to not use @html.